packages/type-utils/tests/TypeOrValueSpecifier.test.ts

```diff
@@ -139,7 +139,8 @@ describe('TypeOrValueSpecifier', () => {
         .program!.getTypeChecker()
         .getTypeAtLocation(
           services.esTreeNodeToTSNodeMap.get(
-            (ast.body[0] as TSESTree.TSTypeAliasDeclaration).id,
+            (ast.body[ast.body.length - 1] as TSESTree.TSTypeAliasDeclaration)
+              .id,
```

Comment:
I actually think this fixes a minor issue that is probably present in more tests - I have taken this directly out of some other tests in this repo